kong/templates/kong_defaults.lua

```diff
@@ -14,6 +14,10 @@ proxy_listen = 0.0.0.0:8000 reuseport backlog=16384, 0.0.0.0:8443 http2 ssl reus
 stream_listen = off
 admin_listen = 127.0.0.1:8001 reuseport backlog=16384, 127.0.0.1:8444 http2 ssl reuseport backlog=16384
 status_listen = off
+cluster_listen = 0.0.0.0:8002
```

Comment:
as it uses websockets, perhaps it is a good idea to add `so_keepalive` option.

Reply:
This is fine as we have application layer heartbeats for the WebSocket connections.